odoo/addons/base/data/res_country_data.xml

```diff
@@ -13,8 +13,10 @@
             <field name="name">United Arab Emirates</field>
             <field name="code">ae</field>
             <field file="base/static/img/country_flags/ae.png" name="image" type="base64" />
+            <field eval="'%(street)s\n%(street2)s\n%(state)s\n%(country_name)s'" name="address_format"/>
```

Comment:
@mpa-odoo  use %(state_name)s or %(state_code)s not %(state)s

Reply:
@jva-odoo  Done